Client/Frontend/Browser/Helpers/LaunchHelper.swift

```diff
@@ -64,4 +94,16 @@ public actor LaunchHelper {
       await AdBlockEngineManager.shared.startTimer()
     }
   }
+  
+  private func measuredTask(label: String, callback: () async -> Void) async {
+    #if DEBUG
+    let startTime = CFAbsoluteTimeGetCurrent()
+    #endif
+    
+    await callback()
+    
+    #if DEBUG
+    print([label, "\(CFAbsoluteTimeGetCurrent() - startTime)"].joined(separator: ": "))
+    #endif
```

Comment:
We could also just use OS Signposts instead.

Reply:
I will remove this logging for now and will wait for you to get back to get this done right.